daemon/mgr/container.go

```diff
@@ -796,8 +796,20 @@ func (mgr *ContainerManager) Restart(ctx context.Context, name string, timeout i
 	}
 
 	logrus.Debugf("start container %s when restarting", c.ID)
+
+	//let restartCount++
```

Comment:
actually, we should add container lock here when we visit container object

Reply:
that's not a problem, i think.

Reply:
that's not a problem, i think